uniffi_bindgen/src/templates/macros.rs

```diff
@@ -6,14 +6,8 @@
 {{ func.name() }}({% call _arg_list_rs_call(func) -%})
 {%- endmacro -%}
 
-{%- macro to_rs_call_with_prefix(prefix, func) -%}
-    {{ func.name() }}(
-    {{- prefix }}{% if func.arguments().len() > 0 %}, {% call _arg_list_rs_call(func) -%}{% endif -%}
-)
-{%- endmacro -%}
```

Comment:
...and now we can get rid of this "call with prefix thing" in the Rust codegen macros! Because the `Object` type now lifts and lowers in the same way as the other types, we don't need to do any specially-structured codegen for it, and can just use the plain `to_rs_call` macro as long as we pass it the full list of arguments.

Reply:
😎